Source/relay/TourGuide/Items of the Month/2024/Chest Mimic

```diff
@@ -0,0 +1,26 @@
+//2024
```

Comment:
Filename should have a .ash extension and needs to be added to the end of `Items of the Month import.ash`